supports/performance-test/compression-test/test_compress_read_performance.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+# avoid blocking REST API
+unset http_proxy
+# for python use
+export PYTHONPATH=../integration-test:$PYTHONPATH
+
+echo "Get configuration from config."
+source config
+echo "------------------ Your configuration ------------------"
+echo "SSM home is ${SMART_HOME}."
+echo "PAT home is ${PAT_HOME}."
+echo "Hadoop home is ${HADOOP_HOME}"
+echo "HiBench home is ${HiBench_HOME}"
+echo "Test case:"
+for size in ${!CASES[@]}; do
+  echo ${size} ${CASES[$size]}
+done
+echo "--------------------------------------------------------"
+
+bin=$(dirname "${BASH_SOURCE-$0}")
+bin=$(cd "${bin}">/dev/null; pwd)
+log="${bin}/ssm.log"
+# remove historical data in log file
+printf "" > ${log}
+
+echo "------------------Generate test data--------------------"
+su hdfs -c "sh ${bin}/prepare.sh"
+echo "--------------------------------------------------------"
+
+
+# Test DFSIO and Terasort continuously for 3 rounds
+for size in "${!CASES[@]}"; do    
+    case="${size}_${CASES[$size]}"
+
+    # Read TestDFSIO
+        action="readDFSIO"
+	echo "Test case ${case}($action):" >> ${log}
+        echo "==================== test case: $action ============================"
+	sh drop_cache.sh
+        # make ssm log empty before test
+        printf "" > ${SMART_HOME}/logs/smartserver.log
+        cd ${PAT_HOME}/PAT-collecting-data
+        echo "su hdfs -c \"hadoop jar ${HADOOP_HOME}/hadoop-mapreduce/hadoop-mapreduce-client-jobclient-*-tests.jar \
```

Comment:
It looks that the test cmds are executed by hdfs user, which is environment dependent. Please add a comment above this line, like "hdfs user will execute the cmd, you can change it to the one in your test env. with execution permission"

Reply:
Thank you for you suggestions. I will do a new pull request after completing these modifications.